bin/configs/scala-http4s.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: scala-http4s
+outputDir: samples/client/petstore/scala-http4s
```

Comment:
thanks for the PR

can you please also add the new folder to https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-scala.yaml so that CI will test is moving forward?

Reply:
@wing328 done. Thank you for the review.